content/community/code.md

```diff
@@ -0,0 +1,31 @@
+# Community Code
+
+Here is a curated list of community created repositories, libraries, components, projects, and code that use Pulumi. If you believe something is missing, send us a [pull request](https://github.com/pulumi/docs/edit/master/content/community/code.md). 
+
+
+## Frameworks/Platforms
+[webiny/webiny-js](https://github.com/webiny/webiny-js) - Platform for building serverless applications and APIs
+
```

Comment:
if you want extra space between the items can we do this with css instead of blank lines?